lib/srv/uacc/uacc.h

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
nit: this file should have the extension `.c`

Reply:
I don't think I agree. It's meant to be a header library consumed by one Go file. Renaming it to C causes go build to attempt to compile and link it separately. Taking this approach would require yet another header .h file that both the .c and .go files can consume. I find it much easier just to stick with the single-header-library approach here.

Reply:
gotcha, let's keep it as is then